src/pages/Tournament/TournamentDetails/AdminManagementPanel/parts.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import { Button, Modal, Select } from 'antd';
+
+import { ClosableComponentProps } from 'utils/interfaces';
+import useFetch from 'hooks/useFetch';
+import { MessageResponse } from 'utils/apiResponseShapes';
+import useToastPushSubmit from 'hooks/useToastPushSubmit';
+import { useTournamentInfoContext } from '../TournamentInfoContext';
+import { useTournamentAdminContext } from '../TournamentAdminContext';
```

Comment:
Codacy has a fix for the issue: [Replace `AdminContext·}·from·'../TournamentAdmin` with `InfoContext·}·from·'../TournamentInfo`](https://app.codacy.com/gh/othello-organisation-singapore/othello-storm-system-frontend/pullRequest?prid=8062249)
```suggestion
import { useTournamentInfoContext } from '../TournamentInfoContext';
```